src/main/java/com/khadri/jdbc/data/dao/util/JdbcConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.khadri.jdbc.data.dao.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class JdbcConnectionUtil {
+
+	private static Connection con;
+
+	public static Connection getConnection() {
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
read from property file and pass to DriverManager class to get the connection. refer the program how to load props in to java in JDBCApss repository.

Reply:
read property file and passed